### ISSUE TITLE
smartling-cli init does not work on Windows

### ISSUE THREAD

**Issue by lhu-smartling** [Smartling/smartling-cli#62]
When running the command `smartling-cli init` on Windows, it does not create a yml file.

**Comment by dimitrystd** [comment]
Yes, known issue. We introduced this bug when implemented #39. I was sure that we already have this bug but looks like your ticket is first.
